operations/research-and-development/product/product-planning/how-to-use-productboard.md

```diff
@@ -52,42 +52,43 @@ In this view, you can filter and sort by various fields, and create custom views
 
-### 1c - Organizing features in the master feature list
+### 1c Organizing features in the Master Feature list
 
-1. Search for your roadmap features and add any that are missing under the appropriate category to the master feature list.
+1. Search for your roadmap features and add any that are missing under the appropriate category to the Master Feature list.
 2. Make sure all features that fall under your area of responsibility are assigned to you.
 3. Add any important detail about the feature that is missing or that would be valuable to others that might need to help you validate or gather requirements for the feature. 
-4. Assign Releases: Features can have one or many releases assigned as they may appear on different roadmap/planning views for different purposes. Below is an outline of our current structure of releases in Product Board:
- - **Planning Estimates**: H1 Plan, H2 Plan (tentative), Backlog, Community Contract, or Bounty
- - **Exact Releases** (when known): Examples: 5.22, 5.24, 1.30, Mattermost 6.0
- - **Sequence**: Now, Next, Later
-5. Setting the Status:  There are 6 statuses throughout the life of a feature:
- - **Idea**: When you have a single request or an idea for a feature it starts out in this status. As new customer insights come in, you will be able to attach them to this idea to validate and measure demand. 
- - **Under Consideration**: This status is used to identify features that are demand validated, but we remain undecided if we will build them (for various reasons).
- - **Planned**: Features that we have validated and decided to build are planned. These features should all have a release assigned (if scheduling is unknown, assign to the release called `backlog`. 
- - **In Development**: All features actively being worked on by UX or R&D belong to this status. 
- - **Postponed**: If a feature is cut from a release, followers of the feature need to be notified. Currently, productboard only notifies followers when a status changes or they are explicitly mentioned. We have added this status to ensure followers of a feature can stay informed when features are cut. 
- - **Won't Do**: This is a status often used after the `Under Consideration` status if we decide for any reason not to build a feature that is demanded by the market. It's important to keep track of for future cases of demand. If you set a feature to this status, please be sure to provide the reason we won't do it. This reason should be in a format that can be shared with propspects/customers. 
+4. **Assign Releases:** Features can have one or many releases assigned as they may appear on different roadmap/planning views for different purposes. Below is an outline of our current structure of releases in productboard:
+ * **Planning Estimates:** H1 Plan, H2 Plan (tentative), Backlog, Community Contract, or Bounty.
+ * **Exact Releases:** (when known) Examples: 5.22, 5.24, 1.30, Mattermost 6.0.
+ * **Sequence:** Now, Next, Later.
```

Comment:
This has changed to 3 months, 6 months, future candidates in the UI but not sure if this point is a reference to that or our own sequence.

Reply:
this has changed a bit. we are no longer doing planning estimates since we have moved that to the portal views with 3mo, 6mo, future candidates estimations.
this line can be removed: `Planning` Estimates: H1 Plan, H2 Plan (tentative), Backlog, Community Contract, or Bounty`